packages/remirror__messages/src/extension-bold-messages.ts

```diff
@@ -1,14 +1,13 @@
-import type { MessageDescriptor } from '@lingui/core';
-import { defineMessage } from '@lingui/macro';
+import type { RemirrorMessage } from '@remirror/core-types';
 
-export const LABEL: MessageDescriptor = defineMessage({
+export const LABEL: RemirrorMessage = /*i18n*/ {
```

Comment:
Why do we need `/*i18n*/` here?

Reply:
This is how you can mark a message for Lingui extraction, without using their `defineMessaage` API

https://lingui.dev/guides/message-extraction#explicitly-marking-messages